docs/source/customizations/overview.mdx

```diff
@@ -28,25 +34,24 @@ sequenceDiagram
 
     Client->>RouterService: request
     RouterService->>QueryPlannerService: plan
-    QueryPlannerService-->>RouterService: 
+    QueryPlannerService-->>RouterService:
```

Comment:
My editor auto-trims whitespace, which is frustrating here because the _lack_ of whitespace breaks the diagram. Ideally I can work out another way to do this two-way arrow arrangement without a trailing space being key to it all.

Reply:
Unfortunately I did not find a way around this. Only way was to add a real label to the responses, but this ended up looking much worse.